UI/window-basic-main.cpp

```diff
@@ -410,6 +410,29 @@ void OBSBasic::UpdateVolumeControlsDecayRate()
 	}
 }
 
+void OBSBasic::UpdateVolumeControlsPeakMeterType()
+{
+	uint32_t peakMeterTypeIdx = config_get_uint(basicConfig, "Audio",
+			"PeakMeterType");
+
+	enum obs_peak_meter_type peakMeterType;
+	switch (peakMeterTypeIdx) {
+	case 0:
+		peakMeterType = SAMPLE_PEAK_METER;
+		break;
+	case 1:
+		peakMeterType = TRUE_PEAK_METER;
+		break;
+	default:
+		peakMeterType = SAMPLE_PEAK_METER;
+		break;
```

Comment:
A `break;` statement is not required for the last case.

Reply:
I consulted with Jim on this one.  He said it doesn't particularly matter, though it's probably better to have it just as a programming safety measure.

Since you probably got an email notification for the original comment, I'm going to leave it here to avoid confusion.